src/megha/Megha_Assignment23.java

```diff
@@ -0,0 +1,191 @@
+package megha;
+
+import java.util.Arrays;
+
+public class Megha_Assignment23 {
+
+	int findMax(int[] arr) {
+		int max = 0;
+		int tempIndex = 0;
+		for (int index = 1; index < arr.length; index++) {
```

Comment:
Check this logic, this will be a good one, you started from index 1, without giving arr[0] to max variable.
![image](https://user-images.githubusercontent.com/46770978/149812762-82bbe051-dc96-4384-a874-eeeca5fbd9a3.png)